collectors/python.d.plugin/alarms/README.md

```diff
@@ -0,0 +1,56 @@
+<!--
+title: "Alarms"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/alarms/README.md
+-->
+
+# Alarms - graphing Netdata alarm states over time
+
+This collector creates an 'Alarms' menu with one line plot showing alarm states over time. Alarm states are mapped to integer values according to the below default mapping. Any alarm status types not in this mapping will be ignored (Note: This mapping can be changed by editing the `status_map` in the `alarms.conf` file).
+
+```
+{
+    'CLEAR': 0, 
+    'WARNING': 1, 
+    'CRITICAL': 2
+}
+```
```

Comment:
I think the link to the [Alarm statuses docs section](https://learn.netdata.cloud/docs/agent/health/reference#alarm-statuses) would be useful here.